content/posts/2020/launching-reana-blog.md

```diff
@@ -0,0 +1,11 @@
+---
+title: "Launching REANA Blog 🚀"
```

Comment:
(1) I get the following locally:

```console
$ hugo server -D
Start building sites … 
WARN 2021/01/11 14:05:15 found no layout file for "HTML" for kind "page": You should create a template file which matches Hugo Layouts Lookup Rules for this combination.
WARN 2021/01/11 14:05:15 found no layout file for "HTML" for kind "section": You should create a template file which matches Hugo Layouts Lookup Rules for this combination.
WARN 2021/01/11 14:05:15 found no layout file for "HTML" for kind "taxonomy": You should create a template file which matches Hugo Layouts Lookup Rules for this combination.
WARN 2021/01/11 14:05:15 found no layout file for "HTML" for kind "taxonomy": You should create a template file which matches Hugo Layouts Lookup Rules for this combination.
WARN 2021/01/11 14:05:15 found no layout file for "HTML" for kind "term": You should create a template file which matches Hugo Layouts Lookup Rules for this combination.
Built in 6 ms
Error: Error building site: failed to render pages: render of "home" failed: ".../blog.reana.io/layouts/index.html:1:3": execute of template failed: template: index.html:1:3: executing "index.html" at <partial "header" .>: error calling partial: partial "header" not found
```

(2) The text content suggestions:

The title capitalisation should be "Launching REANA blog" following British English style, i.e. only first word is capitalised.
BTW we may want to turn it into "REANA blog is launched!".

The content should mention FQDN so that the web site is ensured to be accessible, and perhaps take the occasion to point to forum, so something like:

```
# Launching REANA blog

We are glad to announce the launch of a new blog series about [REANA
reproducible analysis platform](https://www.reana.io). We plan to
describe new REANA releases, share exciting features and other useful
tips and tricks for running containerised data analyses on REANA. 

You can subscribe to the [REANA blog feed](FIXME) or get in touch at
the [REANA user forum](https://forum.reana.io).

Stay tuned for the first blog post!
```

(3) Can you please rebase against latest `master`?

Reply:
(2) and (3) done.

Reply:
New instructions added to README to solve (1).